install/operator/deploy/manifests/1.7.0/syndesisoperator.1.7.0.clusterserviceversion.yaml

```diff
@@ -240,8 +240,7 @@ spec:
               serviceAccountName: syndesis-operator
               containers:
                 - name: syndesis-operator
-                  image: lgarciaac/syndesis-operator:latest
-                  #image: syndesis/syndesis-operator:1.7.1-20190607
+                  image: syndesis/syndesis-operator:latest
```

Comment:
Bit odd that we use `latest` on a versioned manifest

Reply:
The final version for 1.7 is not yet build

Reply:
Perhaps we should tag the latest `1.7.*` with `1.7`, I'm a bit afraid that we'll forget to change this from `latest`.